### DISCUSSION TITLE
PR in Pending State

### DISCUSSION THREAD

**Discussion by Shariq2003** [code-a2z/code-a2z#903] [General]
I have raised a PR that requires some changes according to guidelines and for sure I have update that but still my PR is in pending state facing conflict(Readme.md Update) by the merging of other PRs.

@Avdhesh-Varshney Sir please look into it.

**Comment by Avdhesh-Varshney** [comment]
I've reviewed them. 
- You have to update your project readme file according to readme template, as alignment is distorted.
- Enlist your project in the upper row. Don't move your code up and down only. Review other rows.

**Reply by Shariq2003** [reply]
> I've reviewed them.
> 
> * You have to update your project readme file according to readme template, as alignment is distorted.
> * Enlist your project in the upper row. Don't move your code up and down only. Review other rows.

I have updated sir

**Reply by Shariq2003** [reply]
please check

**Reply by Shariq2003** [reply]
Now i have made the changes according to you in both PRs please review sir
